### PR TITLE
Forward back buttons

### PR DESCRIPTION
**Pull request by ChrisMGeiger**
Implemented forward and back buttons on hosts, hosts services and issues pages. This is implemented with next and prev routes within the router.

The next/prev hotkey code was refactored to reduce redundancy. This code could probably be reworked again to make use of the new routing logic.

Suggestions are welcome how to remove any redundant code or improve the UI components.

### CODE REVIEW

File: app/lib/router/services.js [hunk 1]
```diff
@@ -26,6 +26,30 @@ Router.route('/projects/:id/hosts/:hid/services/:sid', {
...
+    this.redirect('/projects/' + next.projectId + '/hosts/' + next.hostId + '/services/' + next.serviceId + '/' + this.params.page, {}, {replaceState: true})
+    this.next()
+  }
+})
+
+Router.route('/projects/:id/hosts/:hid/services/:sid/:page/prev', {
+  name: 'prevService',
+  controller: 'ProjectController',
+  onBeforeAction: function () {
+    console.log("Executing service prev route")
```
Comment (tomsteele) [inline app/lib/router/services.js]: Can we remove console.log here?

File: app/lib/router/services.js [hunk 2]
```diff
@@ -26,6 +26,30 @@ Router.route('/projects/:id/hosts/:hid/services/:sid', {
   }
 })
 
+
+Router.route('/projects/:id/hosts/:hid/services/:sid/:page/next', {
+  name: 'nextService',
+  controller: 'ProjectController',
+  onBeforeAction: function () {
+    console.log("Executing service next route")
```
Comment (tomsteele) [inline app/lib/router/services.js]: Can we remove console.log here?

Comment (ChrisMGeiger) [inline app/lib/router/services.js]: Yes, forgot to remove that debugging. Will upload a new commit tonight.

**Comment by ChrisMGeiger** [comment]
Left over debugging console.log statements removed.

**Comment by ChrisMGeiger** [comment]
This commit will hopefully resolve issue #75

**Pull request merged**